packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -611,6 +635,26 @@ export default decorate([
               }
             : setting
         ),
+      displayAdvancedSettings: (
+        {
+          _displayAdvancedSettings,
+          compression,
+          isFull,
+          propSettings,
+          settings = propSettings,
+        },
+        { job }
+      ) =>
+        defined(
+          _displayAdvancedSettings,
+          Object.keys(
+            extractSettingsToDisplay({
+              compression: defined(compression, job.compression),
+              mode: isFull ? 'full' : 'delta',
+              settings: { '': settings.get('') },
+            })
+          ).some(setting => setting !== 'reportWhen')
```

Comment:
Too complicated, I don't understand what it does.

Reply:
It looks if it have a setting to display except `reportWhen`.

Reply:
Use `!isEmpty()`